apm-agent-plugins/apm-httpclient-core/src/test/java/co/elastic/apm/agent/httpclient/AbstractHttpClientInstrumentationTest.java

```diff
@@ -169,7 +169,7 @@ protected Span verifyHttpSpan(String host, String path, int status, boolean requ
 
         Http httpContext = span.getContext().getHttp();
 
-        assertThat(span.getNameAsString()).isEqualTo(String.format("%s %s", httpContext.getMethod(), host));
+//        assertThat(span.getNameAsString()).isEqualTo(String.format("%s %s", httpContext.getMethod(), host));
```

Comment:
[question] how is it related to other changes here ?

Reply:
I have no idea 😅

Reply:
I remember now: this was from an abandoned experiment to remove `co.elastic.apm.agent.okhttp.OkHttpClientHelper#computeHostName`
```suggestion
        assertThat(span.getNameAsString()).isEqualTo(String.format("%s %s", httpContext.getMethod(), host));
```